src/dpr/components/data-table/init.js

```diff
@@ -6,10 +6,24 @@ $('[data-navigate-to]').each((index, element) => {
   })
 })
 
+function showNoResultsMessage() {
```

Comment:
Does this need to be a client-side script? Would it be possible to achieve the same thing server-side?

I ask because the machines on site are largely thin clients, so the more we do server-side the better for them.